tensorflow/python/client/timeline.py

```diff
@@ -607,7 +617,48 @@ def _show_memory_counters(self):
                                         total_bytes)
     self._allocator_maximums = alloc_maxes
 
-  def analyze_step_stats(self, show_dataflow=True, show_memory=True):
+  def _preprocess_op_time(self, op_time):
+    """Modified the start and end of ops in step stats"""
```

Comment:
Can you add args to the doc string?

Reply:
I have modified the comment for this function. Thank you.